python/test/unit/language/test_core.py

```diff
@@ -1347,13 +1347,7 @@ def test_atomic_rmw(op, dtype_x_str, mode, sem, device):
     if is_interpreter():
         if dtype_x_str == 'float16':
             pytest.skip("Only test atomic float16 ops on GPU")
-    else:
-        check_cuda_or_hip(device)
 
-    capability = torch.cuda.get_device_capability()
```

Comment:
Remove as suggested in https://github.com/triton-lang/triton/pull/3946/files#r1606944957.